### PR TITLE
added user profile page

### PR DESCRIPTION
**Pull request by JanhaviDadhania**
#48 
go to al.html and login to dashboard
click profile and you will be directed to profile.php

- it displays data from user_info table
- gives an error if anyone directly tries to access the page

### CODE REVIEW

File: CSE-Hub/user/profile.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+$row = mysqli_fetch_row($result);
+
+
+?>
+
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Dashboard</title>
```
Comment (harshraj22) [inline CSE-Hub/user/profile.php]: Maybe this needs to be changed to Profile ?

File: CSE-Hub/user/profile.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
+<?php
+session_start();
+require_once '../sql_login/login.php';
+
+// in case the user directly want's to access this page, php doesn't know whose profile to display
+if(!isset($_GET['pass'])){
```
Comment (harshraj22) [inline CSE-Hub/user/profile.php]: Maybe giving a meaningful variable name would be more helpful (something like username , I hope you are not sending password via $_GET[ ] ).

**Comment by JanhaviDadhania** [comment]
'pass' because I am passing the variable.
Anyways changed it...
and made other changes.

**Review by harshraj22 — APPROVED** [review]
LGTM

**Comment by JanhaviDadhania** [comment]
#47 added pie chart 
displays information from user_submissions table.
result column values in table can be : "WA", "AC", "TLE", "RE", "PA"

**Comment by DestroyerAlpha** [comment]
Looks OK.

**Review by DestroyerAlpha — COMMENTED** [review]
Looks fine.

File: CSE-Hub/user/profile.php [hunk 3]
```diff
@@ -77,13 +119,31 @@
...
+        data.addColumn('string', 'language');
+        data.addColumn('number', 'Nos');
+		for(i = 0; i < my_2d.length; i++)
+    data.addRow([my_2d[i][0], parseInt(my_2d[i][1])]);
+    var options = {title:'stats',
+                       width:600,
+                       height:500};
+    var chart = new google.visualization.PieChart(document.getElementById('chart_div'));
+    chart.draw(data, options);
+    }
```
Comment (harshraj22) [inline CSE-Hub/user/profile.php:147]: How about adding some resources to learn these in [Resources.md](https://github.com/harshraj22/contest/blob/develop/resources/resources.md) ?

Comment (JanhaviDadhania) [reply CSE-Hub/user/profile.php:147]: I will add...

**Pull request merged**